core/pta/bcm/elog.c

```diff
@@ -155,8 +156,10 @@ static TEE_Result pta_elog_nitro_crash_dump(uint32_t param_types,
 		return TEE_ERROR_ACCESS_DENIED;
 	}
 
+	sz = (params[0].memref.size > DEFAULT_ELOG_BUFFER_SIZE) ?
+			DEFAULT_ELOG_BUFFER_SIZE : params[0].memref.size;
 	src_vaddr = (vaddr_t)phys_to_virt((uintptr_t)src_paddr + offset,
-					  MEM_AREA_RAM_SEC);
+					  MEM_AREA_RAM_SEC, sz);
```

Comment:
We should probably feed `DEFAULT_ELOG_BUFFER_SIZE` instead of `sz` here. Please check the return value also.

Reply:
Seems that sz should be used in cache_op_inner() below...

Reply:
Possibly, but I'd say that's out of scope for this PR.

Reply:
Is this means that it is better to leave cache_op_inner() as it was before?

Reply:
Yes